### PR TITLE
Add option to only run play or smoke tests

### PR DESCRIPTION
**Pull request by gabrielperales**
Just a rebase over @josephnle PR to update [his code](https://github.com/storybookjs/test-runner/pull/81).

### CODE REVIEW

File: src/playwright/transformPlaywright.ts [hunk 1]
```diff
@@ -80,6 +80,7 @@ export const transformPlaywright = (src: string, filename: string) => {
     insertTestIfEmpty: true,
     clearBody: true,
     makeTitle: makeTitleFactory(filename),
+    onlyType: process.env.ONLY_TYPE as StorybookTestType,
```
Comment (gabrielperales) [inline src/playwright/transformPlaywright.ts:83]: This is the only thing I'm not sure... 🤔

**Comment by yannbf** [comment]
Hey there, thanks for your contribution (as followup from @josephnle's one). Even though it's a good idea, I was wondering if we could fix this behavior differently, and just use the native test filtering functionality from jest (testNamePattern). Please check this comment for context: https://github.com/storybookjs/test-runner/issues/281#issuecomment-1518251704

**Comment by rebeccadavidsson** [comment]
Any updates on this @gabrielperales ? Would be a very nice feature!

**Comment by gabrielperales** [comment]
@rebeccadavidsson I'm no longer working with storybook tests, so I stopped working on this.

**Comment by yannbf** [comment]
Hey there! I'll be closing this PR because this functionality is now available via test filtering with tags.

for running only play tests
```
yarn test-storybook --includeTags="play-fn"
```

for running only smoke tests
```
yarn test-storybook --excludeTags="play-fn"
```

Thank you!!

**Pull request closed without merging**